etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example

```diff
@@ -0,0 +1,69 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="ALL"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+export EDITOR="nano"
+
+# load packages
+spack unload
+
+# PIConGPU build dependencies #################################################
+#   need to load correct cmake and gcc to compile picongpu
+
+spack load gcc@10.2.0
+spack load cmake@3.23.1 %gcc@10.2.0
+
+# General modules #############################################################
+#   correct dependencies are automatically loaded, if successfully installed using install.sh
+#   and no name confilcts in spack, see install.sh for more precise definition
+#   if name conflicts occur
+
+spack load openpmd-api@0.14.4 %gcc@10.2.0 \
+    ^adios2@2.8.0 \
+    ^hdf5@1.12.1 \
+    ^openmpi@4.1.3 +atomics +cuda cuda_arch=70 \
+    ^python@3.9.12 \
+    ^py-numpy@1.22.3
+spack load boost@1.78.0 +math +filesystem +system %gcc@10.2.0
+
+# PIConGPU output dependencies ################################################
+#
+spack load pngwriter@0.7.0 %gcc@10.2.0
+
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
+export PIC_BACKEND="omp2b:native" # running on cpu
```

Comment:
Should be 

```
export PIC_BACKEND="cuda:70" # running on gpu
```

Reply:
I completely overlooked that. Now everything should be done.